docs/developer/developer.adoc

```diff
@@ -31,7 +31,7 @@ mkdir -p build
 .. Use fresh sources from git repository.
 +
 ----
-$ git clone https://github.com/OpenSCAP/openscap.git
+$ git clone --recurse-submodules https://github.com/OpenSCAP/openscap.git
```

Comment:
Is it worth adding instructions for those who already have a clone checked out?
I had to do `git submodule update --init --recursive` for git to actually fetch the submodule.

Reply:
I would prefer to simply document the fact that we are using git submodules in our project.

Reply:
Makes sense.